app/group/models/fine.py

```diff
@@ -0,0 +1,75 @@
+import uuid
+
+from django.core.exceptions import ValidationError
+from django.db import models
+
+from app.common.enums import AdminGroup
+from app.common.permissions import BasePermissionModel, check_has_access
+from app.content.models.user import User
+from app.group.models.group import Group
+from app.util.models import BaseModel
+
+
+class Fine(BaseModel, BasePermissionModel):
+
+    access = AdminGroup.admin()
+    id = models.UUIDField(
+        auto_created=True, primary_key=True, default=uuid.uuid4, serialize=False,
+    )
+
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name="fines")
+    created_by = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="fines_created", default=None
+    )
+    group = models.ForeignKey(Group, on_delete=models.CASCADE, related_name="fines")
+    amount = models.IntegerField(default=1)
+    approved = models.BooleanField(default=False)
+    payed = models.BooleanField(default=False)
+    description = models.TextField(default="", blank=True)
+    reason = models.TextField(default="", blank=True)
+
+    class meta:
+        verbose_name_plural = "Fines"
+        ordering = ("created_at",)
+
+    def clean(self):
+        if not self.user.is_member_of(self.group):
+            ValidationError("Du er ikke medlem av denne gruppen")
+
+    def save(self, *args, **kwargs):
+        self.full_clean()
+        return super(Fine, self).save(*args, **kwargs)
```

Comment:
Kan bruke nyere syntax her
```suggestion
        return super().save(*args, **kwargs)
```